### PR TITLE
add mainnet v9 proposal, add chain registry files for future automation

### PR DESCRIPTION
**Pull request by 2xburnt**
## Description

xion-mainnet-1 v9 Software Upgrade

## Type of Change

- [x] Governance Proposal
- [x] Software Upgrade
- [ ] Parameter Change
- [ ] New Validator
- [ ] Bug fix
- [ ] Documentation update
- [ ] Tooling enhancement

## Testing

- [x] I've read [CONTRIBUTING.md](../CONTRIBUTING.md)

## Additional Information

Any additional context that you want to provide.

## Reviewers:

/assign @froch
/assign @2xburnt

Thank you for supporting the Burnt Networks!

### CODE REVIEW

File: mainnet/xion-mainnet-1/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![Burnt banner](https://files.xion-testnet-1.burnt.com/banner.jpg)
+![Burnt banner](https://files.xion-mainnet-1.burnt.com/banner.jpg)
```
Comment (froch) [inline mainnet/xion-mainnet-1/README.md]: this doesn't exist. revert to https://files.xion-testnet-1.burnt.com/banner.jpg

**Review by froch — DISMISSED** [review]
✅ Nice work. Left a comment about a dead link to be reverted

File: mainnet/xion-mainnet-1/README.md (hunk 1 above)
Comment (2xburnt) [reply mainnet/xion-mainnet-1/README.md]: fixed!!

**Review by froch — APPROVED** [review]
✅

**Pull request merged**